stream_alert_cli/terraform/cloudtrail.py

```diff
@@ -33,8 +33,10 @@ def generate_cloudtrail(cluster_name, cluster_dict, config):
     cloudtrail_module = 'cloudtrail_{}'.format(cluster_name)
 
     enabled_legacy = modules['cloudtrail'].get('enabled')
-    cloudtrail_enabled = modules['cloudtrail'].get('enable_logging')
-    kinesis_enabled = modules['cloudtrail'].get('enable_kinesis')
+
+    cloudtrail_enabled = modules['cloudtrail'].get('enable_logging', True)
+    kinesis_enabled = modules['cloudtrail'].get('enable_kinesis', True)
```

Comment:
Does this mean we are changing the default values for these configuration options? If so, we probably need to update the documentation

Reply:
No, they should be enabled by default, it's a bug

Reply:
If we don't put a default, Python will marshal None to JSON, which throws Terraform errors